sight_service/worklist_scheduler_opt.py

```diff
@@ -187,71 +166,64 @@ def decision_point(
     # with self.active_lock.gen_wlock():
     #   self.active_samples[request.worker_id] = {'id': key, 'sample': sample}
 
```

Comment:
Do we need this commented code ?

Reply:
its for previous , ref , can remove after stable release